msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -864,129 +864,129 @@ Delegated permissions allow the app to access data on behalf of the signed-in us
-|`tabs.groups.controls.id`| String | 64 characters| ✔️ | Unique identifier for the control within the app. It must be different from any built-in control ID in the Microsoft 365 application and any other custom control. |
+|`tabs.groups.controls.id`| String | 64 characters| ✔️ | Specifies the ID for the control within the app. It must be different from any built-in control ID in the Microsoft 365 application and any other custom control. |
 |`tabs.groups.controls.items`| Object | | | Configures the items for a menu control. |
-|`tabs.groups.controls.items.id`| String | | ✔️ | Unique identifier for the item within the app. |
-|`tabs.groups.controls.items.type`| String enum | | ✔️ | Defines the control item type. Supported value: `menuItem`. |
-|`tabs.groups.controls.items.label`| String | 64 characters| ✔️ | Text displayed for the item. |
+|`tabs.groups.controls.items.id`| String | | ✔️ | Specifies the ID for the items within the app. |
+|`tabs.groups.controls.items.type`| String enum | | ✔️ | Defines the control items type. Supported value: `menuItem`. |
+|`tabs.groups.controls.items.label`| String | 64 characters| ✔️ | Specifies the text displayed for the items. |
 |`tabs.groups.controls.items.icons`| Array | | | Configures the icons for the custom item.|
-|`tabs.groups.controls.items.icons.size`| Number | |✔️| Size of the icon in pixels, enumerated as `16`,`20`,`24`,`32`,`40`,`48`,`64`,`80`. Required image size: `16`, `32`, `80`. |
-|`tabs.groups.controls.items.icons.url`| URL| | | URL of the icon.|
-|`tabs.groups.controls.items.icons.file`| String | 2048 characters| | Relative path to the file that contains the icon. This property isn’t supported as a relative path. |
-|`tabs.groups.controls.items.supertip`| | |✔️| It configures a supertip for the custom item. A supertip is a UI feature that displays a brief box of help information about a control when the cursor hovers over it. The box may contain multiple lines of text. |
-|`tabs.groups.controls.items.supertip.title`| String | 64 characters | ✔️ | Title text of the supertip.|
-|`tabs.groups.controls.items.supertip.description`| String | 128 characters | ✔️ | Description of the supertip.|
-|`tabs.groups.controls.items.actionId`| String | 64 characters | ✔️ | Identifies the action that is taken when a user selects the control or menu item. The *actionId* must be an exact match for a `runtime.actions.id`. |
-|`tabs.groups.controls.items.enabled`| Boolean | | | It indicates whether the control is initially enabled. Default value: `true`.|
+|`tabs.groups.controls.items.icons.size`| Number | |✔️| Specifies the size of the icon in pixels, enumerated as `16`,`20`,`24`,`32`,`40`,`48`,`64`,`80`. Required image size: `16`, `32`, `80`. |
+|`tabs.groups.controls.items.icons.url`| URL| | | Specifies the URL of the icon.|
+|`tabs.groups.controls.items.icons.file`| String | 2048 characters| | Specifies the relative path to the file that contains the icon. This property isn’t supported as a relative path. |
```

Comment:
Update the `url` property to be "Specifies the absolute URL of the icon." 
Remove the `file` property entirely.